src/components/link/index.ts

```diff
@@ -0,0 +1,22 @@
+import Block from '../../utils/Block';
+import template from './link.hbs';
+
+interface LinkProps {
+  href?: string;
+  lable?: boolean;
```

Comment:
по названию можно подумать, что там должна лежать строка. Предлагаю назвать как-то так
```suggestion
  hasLable?: boolean;
```